v2/fleet/datadog-control.service

```diff
@@ -0,0 +1,24 @@
+# https://raw.githubusercontent.com/DataDog/docker-dd-agent/master/dd-agent.service
+# make sure the etcd /ddapikey is set!
+[Unit]
+Description=Datadog Agent for Mesos, Marathon & Zookeeper
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+TimeoutStartSec=0
+ExecStartPre=/usr/bin/docker pull behance/docker-dd-agent-mesos:latest
+ExecStartPre=-/usr/bin/docker kill dd-agent-mesos
+ExecStartPre=-/usr/bin/docker rm -f dd-agent-mesos
+ExecStart=/usr/bin/bash -c \
+"if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && sudo /usr/bin/docker run --name dd-agent-mesos -h `hostname` \
+-e API_KEY=`etcdctl get /ddapikey` \
+-e MESOS_HOST \
```

Comment:
So the control ELB will also need to be set in etcd. Something along the lines of;
```
-e MESOS_HOST=`etcdctl get /ddmesoshost`
```
```bash
chris@ip-10-43-1-13 stage-portfolio-us-west-2a-control ~ $ etcdctl get /ddmesoshost
internal-portfolio-control-elb-name-other-stuff.us-south-8.elb.amazonaws.com
```